.github/workflows/lint-test-build.yml

```diff
@@ -0,0 +1,73 @@
+name: Lint, test and build
+on: [push]
+jobs:
+    cache-dependencies:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v3
+              with:
+                  node-version: ${{ matrix.node-version }}
+                  cache: "npm"
+            - uses: actions/cache@v2
+              with:
+                  path: "**/node_modules"
+                  key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}-${{ hashFiles('**/package.json') }}
+            - name: Installing dependencies
+              run: npm i
```

Comment:
If we do have a package lock, don't we want to run de `npm ci` command here?

Reply:
Yes. Fixed